WizardExtensions/MSTestv2IntelliTestExtension/MSTestv2IntelliTestExtension.csproj

```
@@ -14,6 +14,6 @@
     <Compile Include="MSTestv2TestFrameworkPackageAttribute.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Internal.IntelliTest" Version="$(MicrosoftInternalIntelliTestVersion)"/>
+    <PackageReference Include="Microsoft.VisualStudio.Intellitest.Dependencies." Version="$(MicrosoftInternalIntelliTestVersion)"/>
```

There is probably a typo with the trailing period.

```suggestion
    <PackageReference Include="Microsoft.VisualStudio.Intellitest.Dependencies" Version="$(MicrosoftInternalIntelliTestVersion)"/>
```